elements/pfe-navigation/src/pfe-navigation.html

```diff
@@ -17,11 +17,9 @@
       <div id="pfe-navigation__menu-wrapper" class="pfe-navigation__menu-wrapper">
```

Comment:
@kelsS If you add a `<slot hidden></slot>` here, that will capture all your light DOM and wrap it in the hidden styles.  No altering light DOM mark-up at all needed!  🎊

Reply:
Which `slot`? 🤔 Is this code comment not on the line it was intended for?

Reply:
Yeah GitHub sometimes moves things around a bit.  Add an unnamed slot which will capture all other light DOM added to the component.

Reply:
Gotcha, I intentionally avoided a default slot, we can talk about this in our meeting on Monday about the approach and pros/cons.